DataConnectors/ESETPROTECT/Connector_Syslog_ESETPROTECT.json

```diff
@@ -0,0 +1,154 @@
+{
```

Comment:
Please also move these added files to a new folder under the solutions folder, /Solutions/ESETProtect.

For example of desired solution folder structure see: https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/Box